### ISSUE TITLE
[BUG] component @param missing in the generated documentation.

### ISSUE THREAD

**Issue by aner-andros** [compodoc/compodoc#225]
##### **Overview of the issue**
Component's method parameters missing in the generated documentation (Components section).

anag.component.ts:

```
import { ChangeDetectorRef, Component, OnDestroy, OnInit } from '@angular/core';
import { ActivatedRoute } from '@angular/router';
import { NgForm } from '@angular/forms';
import { PFPerson } from '../../model/pf-person';

@Component({
    selector: 'app-anag',
    templateUrl: './anag.component.html',
    styleUrls: ['./anag.component.scss']
})
export class AnagComponent implements OnInit, OnDestroy {

    public modelA: PFPerson;
    openAccordion: boolean;

    constructor() {
        this.modelA = new PFPerson();
        this.openAccordion = false;
    }
    ngOnInit() {
        /** ngOnInit */
    }
    ngOnDestroy() {
        /** ngOnDestroy */
    }
    /**
     * Handler function for ngModelChange event binding.
     * @param checkBoxValue
     * @param form
     */
    public onChangeD(checkBoxValue: boolean, form: NgForm): void {
	this.openAccordion = !!checkBoxValue;
        form.controls['mgDom'].reset();
    }
}
```

generated documentation:

![anag_doc](https://user-images.githubusercontent.com/30146536/28168395-498265e4-67df-11e7-89d6-be9199bdb545.jpg)

##### **Motivation for or Use Case**

`checkBoxValue ` and `form ` parameters missing in the documentation.

##### **Operating System, Node.js, npm, compodoc version(s)**

Windows 7
Node 6.10.3
Npm 3.7.5
compodoc 1.0.0-beta.12
angular-cli 1.0.0

##### **Angular configuration, a `package.json` file in the root folder**

```
{
  "name": "nfe",
  "version": "1.0.0",
  "license": "MIT",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build -prod",
    "postbuild": "grunt pkg:dist",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e",
    "compodoc": "compodoc -p src/tsconfig.app.json -s"
  },
  "private": true,
  "dependencies": {
    "@angular/common": "=4.0.0",
    "@angular/compiler": "=4.0.0",
    "@angular/core": "=4.0.0",
    "@angular/forms": "=4.0.0",
    "@angular/http": "=4.0.0",
    "@angular/platform-browser": "=4.0.0",
    "@angular/platform-browser-dynamic": "=4.0.0",
    "@angular/router": "=4.0.0",
    "@ngx-translate/core": "^7.0.0",
    "@ngx-translate/http-loader": "^0.1.0",
    "angular2-materialize": "=15.0.4",
    "classlist.js": "=1.1.20150312",
    "core-js": "=2.4.1",
    "intl": "=1.2.5",
    "materialize-css": "=0.99.0",
    "moment": "^2.18.1",
    "mydatepicker": "=2.0.1",
    "rxjs": "=5.1.0",
    "web-animations-js": "=2.2.5",
    "zone.js": "=0.8.4"
  },
  "devDependencies": {
    "@angular/cli": "1.0.0",
    "@angular/compiler-cli": "=4.0.0",
    "@compodoc/compodoc": "^1.0.0-beta.12",
    "@ngui/auto-complete": "=0.13.3",
    "@types/jasmine": "2.5.38",
    "@types/node": "~6.0.60",
    "codelyzer": "~2.0.0",
    "grunt": "^1.0.1",
    "grunt-contrib-clean": "^1.1.0",
    "grunt-war": "^0.5.1",
    "jasmine-core": "~2.5.2",
    "jasmine-spec-reporter": "~3.2.0",
    "karma": "~1.4.1",
    "karma-chrome-launcher": "~2.0.0",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "=0.2.0",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "=0.2.2",
    "load-grunt-tasks": "^3.5.2",
    "protractor": "~5.1.0",
    "ts-node": "~2.0.0",
    "tslint": "~4.5.0",
    "typescript": "~2.2.0"
  }
}
```

##### **Compodoc installed globally or locally ?**

globally 

##### **Reproduce the error**

Running npm script:
`npm run compodoc`

**Comment by aner-andros** [comment]
@vogloblinsky  you reproduce the bug?

**Comment by vogloblinsky** [comment]
@aner-andros no, can you retry with 1.0.0-beta.13 ?
Thanks

**Comment by aner-andros** [comment]
##### **Overview of the issue**
I reproduce the bug also with 1.0.0-beta.13 version: component's method parameters missing in the generated documentation.

##### **Operating System, Node.js, npm, compodoc version(s)**
Windows 7
Node 6.10.3
Npm 3.7.5
compodoc 1.0.0-beta.13
angular-cli 1.0.0

##### **Compodoc installed globally or locally ?**
globally

##### **Reproduce the error**
Running npm script:
`npm run compodoc`

**Comment by vogloblinsky** [comment]
Fixed, coming in next release soon 1.0.0-beta.14
